geopy/geocoders/googlev3.py

```diff
@@ -46,6 +46,7 @@ def __init__(
             timeout=DEFAULT_TIMEOUT,
             proxies=None,
             user_agent=None,
+            channel='',
```

Comment:
because channel is going to be a query parameter; 
an empty parameter is ok, but a None parameter when it is sent out it will be understood as a channel named None